packages/lesswrong/lib/betas.ts

```diff
@@ -45,6 +45,8 @@ export const userHasEagProfileImport = disabled;
 
 export const userHasEAEmojiReacts = isEAForum ? testServerOnly : disabled;
 
+export const userHasElasticsearch = isEAForum ? shippedFeature : disabled;
```

Comment:
I'll disable this before deploying. This will allow us to have the backend running in prod to export the data whilst still using algolia on the frontend, then we can switch to the elastic frontend by updating this again in a separate PR.